test/schemas.test.js

```diff
@@ -33,3 +33,20 @@ describe('Forum post schema test', function() {
         });
     });
 });
+ 
+describe('Comment schema test', function() {
+    it('should be invalid if required fields are empty', function(done) {
+        const f = new forum();
```

Comment:
You need to add the comment schema file to this and change the new forum() to new comment() or whatever you decide to call it.